programs/clearing_house/src/controller/amm.rs

```diff
@@ -358,6 +365,34 @@ pub fn update_pool_balances(
         fraction_for_amm = 0;
     }
 
+    {
+        let amm_fee_pool_token_amount_2 = cast_to_i128(get_token_amount(
+            market.amm.fee_pool.balance(),
+            bank,
+            market.amm.fee_pool.balance_type(),
+        )?)?;
+
+        let bank_transfer_insurance_pool_token_amount =
```

Comment:
does this mean we're sending all of the perp exchage fees that aren't reserved for amm to the if?

Reply:
renaming it revenue pool and allowing scale for portion to IF / potential revenue vault